File: dart/test/exists_test.dart
import 'harness.dart';

import '../lib/BluefinDecryptxP2peSDK.dart';

void tests() {
  describe('exists', () {
    test('test-mode', (t) async {
      final testsdk = BluefinDecryptxP2peSDK.test();
      equal(true, null != testsdk);
    });
  });
}

File: lua/test/exists_test.lua
-- BluefinDecryptxP2pe SDK exists test

local sdk = require("bluefin-decryptx-p2pe_sdk")

describe("BluefinDecryptxP2peSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
